client/src/plugins/zeebe-plugin/deployment-plugin/DeploymentPluginValidator.js

```diff
@@ -309,5 +309,5 @@ function shallowEquals(a, b) {
 }
 
 function validCloudUrl(url) {
-  return /^(https:\/\/|)[a-z\d-]+\.[a-z]+-\d+\.zeebe\.camunda\.io(:443|)\/?/.test(url);
+  return /^(https:\/\/|)[a-z\d-]+\.[a-z]+-\d+\.zeebe\.[\w.]+\.[\w]+(:443|)\/?/.test(url);
```

Comment:
Note that `.` in brackets still matches any character, so this URL is valid according to the updated RegEx:

`https://asdf222.usa-22.zeebe.ultrawombat.ultra-@)(wombat.com:443`

Reply:
What's funny about this example is that `@` introduces basic authentication to the URL so that this part `asdf222.usa-22.zeebe.ultrawombat.ultra-` is the username 🗡️

Reply:
Could probably fixed like that:

```suggestion
  return /^(https:\/\/|)[a-z\d-]+\.[a-z]+-\d+\.zeebe\.([\w-]+\.)+\.[\w]+(:443|)\/?/.test(url);
```

So we allow any number of sequences of `a-z, A-Z, 0-9, -` followed by a dot `.`.

Reply:
Hmm the suggestion seems to be invalid. Sorry.

Reply:
@MaxTru Please have a look now.

Reply:
good catch! I forgot/did not consider that [.] would match any character.

The suggestion makes absolute sense. I will adjust.

Reply:
Hmm I think I got it wrong. Indeed `[.]` matches only a dot. The problem is probably elsewhere...

Reply:
As discussed. I added the `$` at the end to avoid such situations.